pkg/cli-runtime/config.go

```diff
@@ -31,6 +31,8 @@ import (
 
 const defaultTanzuIgnoreFile = ".tanzuignore"
 
+var space = " "
```

Comment:
there is no need  to use a spacer constant, just put the space in the format directly

Reply:
Taken care